modules/ztp-create-factory-hub-cluster.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="preparing-the-factory-install-environment_{context}"]
+= Preparing the factory install environment
+
+.Prerequisites
+Before you can provision edge clusters at scale, you must meet the following prerequisites:
+
+=== Base
+
+* Deploy the {product-title} cluster with three control plane nodes following the guidance in the section "Deploying installer-provisioned clusters on bare metal". Alternatively you can use the technology preview Assisted Installer from link:https://cloud.redhat.com/[cloud.redhat.com] to create the cluster.
+** All Cluster Operators are available.
+** Cluster is reachable using a `KUBECONFIG` file.
+** The API, API-INT and ingress should be deployed on the DHCP external Network (factory network).
```

Comment:
The API, API-INT and ingress are of the edge cluster, right? If yes - need to state it explicitly.

Reply:
I think you are best qualified to comment on that but i guess I should call that out